examples/model_assisted_labeling/image_mal.ipynb

```diff
@@ -23,13 +23,7 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    COLAB = "google.colab" in str(get_ipython())
</pre><p>should we standardize this instead?</p><p> </p><p>↓ used in your other video MAL notebook</p><pre class="ql-syntax" spellcheck="false">API_KEY = os.environ.get("LABELBOX_API_KEY")
if not API_KEY:
&nbsp; &nbsp; raise EnvironmentError("Missing API Key")
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/375/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>wait nvm this</p>